.circleci/config.yml

```diff
@@ -133,6 +136,22 @@ jobs:
             ./cc-test-reporter upload-coverage
           when: on_success
 
+  e2e_tests:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /home/circleci/
+      - run:
+          <<: *install_dependencies
+      - run:
+          name: RUN_E2E_TESTS_FILE_UPLOAD_FEATURES
+          command: |
+            bash scripts/run_e2e_tests.sh --suite="fileUploadFeatures"
+      - run:
+          name: RUN_E2E_TESTS_TOPIC_AND_STORY_EDITOR
+          command: |
+            bash scripts/run_e2e_tests.sh --suite="topicAndStoryEditor"
```

Comment:
Please separate these tests, so they are like the frontend or backend tests. Name them like `e2e_file_upload_features` and `e2e_topic_and_story_editor`.

Reply:
Done.